### PR TITLE
Fixes part of #40 & #42: Highfi buttons part 1

### PR DESCRIPTION
**Pull request by rt4914**
## Explanation

Fixes part of #40
Fixes part of #42

This PR works on the highfi part of `Return to Topic` and `Replay` button. Both of these buttons appear only at the end of question/exploration player session and therefore they can never have a split view. Also, the top-margins are consistent across all combinations and therefore they have been defined only in `values/dimens` and not in any other `dimens` file.

### Replay + Return to Topic Button 

[Exploration Mobile-Portrait](https://xd.adobe.com/view/dfe8ccd4-39eb-414d-a0b2-b5674357f966-8abb/screen/3c9aff8d-2319-4570-818e-cbef229a7419/specs/)
[Exploration Mobile-Landscape](https://xd.adobe.com/view/dfe8ccd4-39eb-414d-a0b2-b5674357f966-8abb/screen/8b4be174-68ae-4b99-b937-9f517c6799d6/)
[Exploration Tablet-Portrait](https://xd.adobe.com/view/dfe8ccd4-39eb-414d-a0b2-b5674357f966-8abb/screen/986ffd25-9e86-42a2-b870-aa6e754799e4/)
[Exploration Tablet-Landscape](https://xd.adobe.com/view/dfe8ccd4-39eb-414d-a0b2-b5674357f966-8abb/screen/9b11d6ac-ba50-4d28-b355-28916363cfc9/)

[Question Mobile-Portrait](https://xd.adobe.com/view/dfe8ccd4-39eb-414d-a0b2-b5674357f966-8abb/screen/b6d8fedf-85aa-4052-9b7a-5d6be7f03e01/)
[Question Mobile-Landscape](https://xd.adobe.com/view/dfe8ccd4-39eb-414d-a0b2-b5674357f966-8abb/screen/acb4086c-cdd8-4d4d-a909-a2d18e1472fc/)
[Question Tablet-Portrait](https://xd.adobe.com/view/dfe8ccd4-39eb-414d-a0b2-b5674357f966-8abb/screen/243dda5c-50b8-4666-b3f2-22bbd7e6023a/)
[Question Tablet-Landscape](https://xd.adobe.com/view/dfe8ccd4-39eb-414d-a0b2-b5674357f966-8abb/screen/01adc357-dec8-4f7a-b639-a9aafa89f3f3/specs/)

### All other buttons (Next/Previous/Submit/Continue)

[Exploration Mobile-Portrait](https://xd.adobe.com/view/dfe8ccd4-39eb-414d-a0b2-b5674357f966-8abb/screen/e6a1bbe6-0975-4900-9267-77d31485716b/)
[Exploration Mobile-Landscape](https://xd.adobe.com/view/dfe8ccd4-39eb-414d-a0b2-b5674357f966-8abb/screen/cba839c4-72fb-4fe6-8687-e6096ae67636/)
[Exploration Tablet-Portrait](https://xd.adobe.com/view/dfe8ccd4-39eb-414d-a0b2-b5674357f966-8abb/screen/dd6204c6-faf7-49b5-bd90-e87537f00715/)
[Exploration Tablet-Landscape](https://xd.adobe.com/view/dfe8ccd4-39eb-414d-a0b2-b5674357f966-8abb/screen/821f957c-93d4-4aec-8bfd-3c68e5498da1/)

[Question Mobile-Portrait](https://xd.adobe.com/view/dfe8ccd4-39eb-414d-a0b2-b5674357f966-8abb/screen/3ebedceb-6911-4260-ab08-5e3247aaa8c3/)
[Question Mobile-Landscape](https://xd.adobe.com/view/dfe8ccd4-39eb-414d-a0b2-b5674357f966-8abb/screen/45009a2a-d86e-4f5f-98b7-18b6c7097ed6/)
[Question Tablet-Portrait](https://xd.adobe.com/view/dfe8ccd4-39eb-414d-a0b2-b5674357f966-8abb/screen/723e0e8e-0da5-4613-88c3-4e155b31e289/)
[Question Tablet-Landscape](https://xd.adobe.com/view/dfe8ccd4-39eb-414d-a0b2-b5674357f966-8abb/screen/bfe0e299-2e0e-48c0-a74c-0b573d44117f/)

[Split View Tablet-Portrait](https://xd.adobe.com/view/dfe8ccd4-39eb-414d-a0b2-b5674357f966-8abb/screen/b4f695e2-9b71-4cac-b323-aec57c0c545c/)
[Split View Tablet-Landscape](https://xd.adobe.com/view/dfe8ccd4-39eb-414d-a0b2-b5674357f966-8abb/screen/a32ccabf-4ae8-4112-a021-333f0d0466d6/)

[Buttons Specs](https://xd.adobe.com/view/dfe8ccd4-39eb-414d-a0b2-b5674357f966-8abb/screen/1f784c1f-a867-4c98-9e55-1ee1e7ad4965/specs/)

## Screenshots

    

    

      

    

=======
=======

                   

=======
=======

                

=======
=======

              

## Checklist

- [x] The PR title starts with "Fix #bugnum: ", followed by a short, clear summary of the changes. (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
- [x] The PR explanation includes the words "Fixes #bugnum: ..." (or "Fixes part of #bugnum" if the PR only partially fixes an issue).
- [x] The PR follows the [style guide](https://github.com/oppia/oppia-android/wiki/Coding-style-guide).
- [x] The PR does not contain any unnecessary auto-generated code from Android Studio.
- [x] The PR is made from a branch that's **not** called "develop".
- [x] The PR is made from a branch that is up-to-date with "develop".
- [x] The PR's branch is based on "develop" and not on any other branch.
- [x] The PR is **assigned** to an appropriate reviewer in both the **Assignees** and the **Reviewers** sections.

### CODE REVIEW

File: app/src/main/res/layout/previous_button_item.xml [hunk 1]
```diff
@@ -17,56 +17,30 @@
...
-    app:explorationSplitViewMarginStart="@{@dimen/previous_button_item_exploration_split_view_margin_start}"
-    app:explorationSplitViewMarginTop="@{@dimen/previous_button_item_exploration_split_view_margin_top}"
-    app:explorationSplitViewPaddingApplicable="@{buttonViewModel.hasConversationView &amp;&amp; buttonViewModel.isSplitView}"
-    app:explorationSplitViewPaddingBottom="@{@dimen/space_0dp}"
-    app:explorationSplitViewPaddingEnd="@{@dimen/space_0dp}"
-    app:explorationSplitViewPaddingStart="@{@dimen/space_0dp}"
-    app:explorationSplitViewPaddingTop="@{@dimen/space_0dp}"
+    app:explorationSplitViewMarginEnd="@{@dimen/general_button_item_exploration_split_view_margin_end}"
+    app:explorationSplitViewMarginStart="@{@dimen/general_button_item_exploration_split_view_margin_start}"
+    app:explorationSplitViewMarginTop="@{@dimen/general_button_item_split_view_margin_top_right}"
```
Comment (rt4914) [inline app/src/main/res/layout/previous_button_item.xml]: Change to left.

File: app/src/main/res/layout/previous_button_item.xml [hunk 2]
```diff
@@ -17,56 +17,30 @@
...
-    app:questionSplitViewMarginStart="@{@dimen/previous_button_item_question_split_view_margin_start}"
-    app:questionSplitViewMarginTop="@{@dimen/previous_button_item_question_split_view_margin_top}"
-    app:questionSplitViewPaddingApplicable="@{!buttonViewModel.hasConversationView &amp;&amp; buttonViewModel.isSplitView}"
-    app:questionSplitViewPaddingBottom="@{@dimen/space_0dp}"
-    app:questionSplitViewPaddingEnd="@{@dimen/space_0dp}"
-    app:questionSplitViewPaddingStart="@{@dimen/space_0dp}"
-    app:questionSplitViewPaddingTop="@{@dimen/space_0dp}"
+    app:questionSplitViewMarginEnd="@{@dimen/general_button_item_question_split_view_margin_end}"
+    app:questionSplitViewMarginStart="@{@dimen/general_button_item_question_split_view_margin_start}"
+    app:questionSplitViewMarginTop="@{@dimen/general_button_item_split_view_margin_top_right}"
```
Comment (rt4914) [inline app/src/main/res/layout/previous_button_item.xml]: Change to left.

**Comment by rt4914** [comment]
@anandwana001 Check the last commit and https://github.com/oppia/oppia-android/pull/3197/commits/3109a12dc4c1975abc6d98073da5706d4ecc27ab and screenshot and that should suffice now.

Thanks.

**Review by anandwana001 — APPROVED** [review]
LGTM, Thanks @rt4914

**Pull request merged**